packages/client/pages/jobs/index.tsx

```diff
@@ -15,9 +16,14 @@ export default function Index() {
         Curated Web3 and blockchain jobs at companies or startups.
       </Text>
       <JobFilter />
-      <Container w="100%" maxW="100%">
-        <JobCard />
-      </Container>
+      <Flex
+        maxW="100%"
+        mt="2.5%"
+        w="100%"
+        wrap="wrap"
+      >
+        <JobCard {...{} as Job} />
```

Comment:
I'm destructuring an empty object mostly to keep Typescript happy as I implemented the `props` as type `Job` but currently aren't getting or passing any data into it. Would love some feedback if this could be improved in some capacity!